installer/install.go

```diff
@@ -0,0 +1,397 @@
+//				}).Error("Docker load error")
+//				return fmt.Errorf("error loading images")
+//			}
+//
+//			log.WithFields(log.Fields{
+//				"status": response.Status,
+//				"id":     response.Id,
+//			}).Info("Pushing image...")
+//		}
+//	}
+//	return nil
+//}
+
+func createNamespace(ns string) (string, error) {
+	cmd := exec.Command("kubectl", "create", "ns", ns)
+	return runCmd(cmd)
+}
+
+func runCmd(cmd *exec.Cmd) (string, error) {
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		return string(output), fmt.Errorf("%s: %s", err, output)
+	}
+	return string(output), nil
+}
+
+type ErrorDetail struct {
+	Message string `json:"message"`
+}
+
+type DockerPushResponse struct {
+	Status      string       `json:"status"`
+	Id          string       `json:"id"`
+	Error       string       `json:"error"`
+	ErrorDetail *ErrorDetail `json:"errorDetail"`
+}
+
+type DockerLoadResponse struct {
+	Stream      string       `json:"stream"`
+	Error       string       `json:"error"`
+	ErrorDetail *ErrorDetail `json:"errorDetail"`
+}
+
+type Options struct {
+	ImageRepo         string `short:"r" long:"image-repo" required:"false" description:"Private Docker image repo URL" value-name:"URL"`
+	ImageTar          string `short:"p" long:"image-tar" required:"false" description:"Path to image tar" value-name:"PATH"`
+	//ImageRepoUser     string `long:"repo-user" required:"false" description:"Private Docker image repo URL" value-name:"USER"`
+	//ImageRepoPw       string `long:"repo-pw" required:"false" description:"Private Docker image repo URL" value-name:"PASSWORD"`
+	ClusterName       string `short:"c" long:"cluster-name" required:"true" description:"Private cluster name" value-name:"NAME"`
+	Token     		  string `short:"t" long:"token" required:"true" description:"Astra API token" value-name:"TOKEN"`
+	AcceptEula        bool   `long:"accept-eula" required:"true" description:"(flag) Accept End User License Agreement"`
+	AstraAccountId    string `short:"a" long:"account-id" required:"true" description:"Astra account ID" value-name:"ID"`
+	AstraUrl          string `short:"u" long:"astra-url" required:"false" default:"https://eap.astra.netapp.io" description:"Url to Astra. E.g. 'https://integration.astra.netapp.io'" value-name:"URL"`
+	SkipTlsValidation bool   `short:"z" long:"disable-tls" required:"false" description:"(flag) Disable TLS validation. TESTING ONLY."`
+	HostAliasIP string `long:"hostAliasIP" required:"false" description:"The IP of the Astra host. TESTING ONLY." value-name:"IP"`
+	HostAlias bool   `long:"hostAlias" required:"false" description:"(flag) Set to enable HostAliasIP. TESTING ONLY"`
+
+}
+
+const (
+	NatsImageName               = "nats"
+	ConnectorDefaultsConfigPath = "./astraconnector_defaults.yaml"
+	YamlOutputPath              = "./deployConfig.yaml"
+	OperatorYamlPath            = "./astraconnector_operator.yaml"
+	ConnectorNamespace 			= "astra-connector"
+	ConnectorOperatorNamespace  = "astra-connector-operator"
+)
+
+func main() {
```

Comment:
There could definitely be a lot more functions being used. I've been focused on just getting it working under the time crunch. I would like to come back and make this prettier when I can